data/human/transport missions.txt

```diff
@@ -3304,3 +3304,199 @@ mission "Spacediving amateurs"
 	on complete
 		payment 30000
 		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."
+
+
+mission "Mafia Extortion"
+	minor
+	source
+		government "Pirate"
+	on accept
+		payment -200000
+		fail
+	to offer
+		random < 2
+		credits >= 2000000
+	on offer
+		conversation
+			`A man in a tailored suit, sunglasses, shiny shoes, and green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`
+			`	You notice that he is missing a finger on each hand.`
+			choice
+				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of talking to?"`
+					goto nameless
+				`	"Yes, I'm very happy with my ship."`
+					goto specifics
+				`	"What should I be afraid of happening to my ship?"`
+					goto specifics
+				`	"You sound like one of those Mafia goons."`
+					goto mafia
+				`	"Are you trying to extort me?"`
+					goto mafia
+				`	"Yes, it helps me transport goods for some of the most dangerous people on this planet."`
+					goto unnerved
+				`	"I don't have time to talk."`
+					goto funeral
+			label nameless
+			`	The man snorts. "Wow, telling everybody you meet your full name. That totally won't end up painting a huge target on your head if said name has a bounty on it. But I digress."`
+			choice
+				`	"Sounds like you're afraid of talking to me. Why is that?"`
```

Comment:
Hmm, I'm not sure the above line of dialogue gives the impression that the dude is afraid of you. It seems like he's dissing you. So this response option doesn't make sense to me.

Reply:
Yes, it worked better with the grumpy reply that we had on line 3339 before. I'll remove this choice.